applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -551,25 +555,11 @@ int unicast_server_config_get(uint32_t *bitrate, uint32_t *sampling_rate_hz,
 	}
 
 	if (bitrate != NULL) {
-		int dur_us;
-
-		ret = le_audio_duration_us_get(audio_streams[0].codec_cfg, &dur_us);
+		ret = le_audio_bitrate_get(audio_streams[0].codec_cfg, bitrate);
 		if (ret) {
-			LOG_ERR("Invalid frame duration: %d", ret);
+			LOG_ERR("Unable to get bitrate: %d", ret);
```

Comment:
Nitpick: Do we need to translate the error here? Can we just do a `return ret`?

Also, the error prints should be the same on line 211 and here.

Reply:
I'm not sure how valuable the error translation is actually. However, all the other `_get()` calls in this function does, and return the same error code, so I think it's better to keep the log message to not make the bitrate a special case within the function.

Updated the log message so it's the same as the other bitrate call.